configs/herkin.default.config.js

```diff
@@ -0,0 +1,36 @@
+const path = require('path')
+
+const {
+  DOC_APP_PATH,
+  HERKIN_TESTS_ROOT,
+  HERKIN_FEATURES_DIR,
+  HERKIN_STEPS_DIR,
+  HERKIN_SUPPORT_DIR,
+  HERKIN_UNIT_DIR,
+  HERKIN_WAYPOINT_DIR,
+} = process.env
+
+const rootDir = path.join(__dirname, '../')
+const dockerTestsRoot = path.join(rootDir, 'tests')
+const hostTestsRoot = HERKIN_TESTS_ROOT || path.join(rootDir, 'repos', 'example', 'tests')
```

Comment:
when keg-herkin is deploy as a module. it wont have the `/repos/example` app right? but it will always have `/rootdir/tests` ?
so should the default point to that instead of `/repos/example/tests`? or na

Reply:
Yah, Daniels correct here. We should point to the `/tests` folder.
The example app should work exactly the same as how other apps use keg-herkin.
By setting this default, we could potentially run into issues.
Keg-Herkin, should know nothing about the example app. It just exists within the repo and that's it

Reply:
agreed, updated to use an empty tests directory